cypress/e2e/home/home.cy.js

```diff
@@ -0,0 +1,131 @@
+const path = require("path");
+const API_URL = Cypress.env('API_URL');
+const CUSTOMERS_URL = `${API_URL}/customers`;
+
+describe('Home Page -', () => {
+  beforeEach(() => {
+    cy.setCookie('cookieConsent', 'accepted');
+    cy.visit('/')
+  })
+
+  it('See welcome message with entered name', () => {
+    cy.get('#name').type('Pedro')
+    cy.get('h2').contains('Hi Pedro!').should('be.visible')
+  })
+
+  context('Customer Details', () => {
+    it.only('It shows and hides the customer address', () => {
```

Comment:
```suggestion
    it('It shows and hides the customer address', () => {
```

Cuidado para não fazer o _commit_ de testes com `.only`.